ceph.spec.in

```diff
@@ -405,7 +417,17 @@ License:	LGPL-2.0
 Requires:	librados2 = %{epoch}:%{version}-%{release}
 Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
 %description -n python-rados
-This package contains Python libraries for interacting with Cephs RADOS
+This package contains Python 2 libraries for interacting with Cephs RADOS
+object store.
+
+%package -n python%{python3_pkgversion}-rados
```

Comment:
I'm concerned that this `%python3_pkgversion` macro might not be defined. Could you add something like

```
%{!?python3_pkgversion: %global python3_pkgversion 3}
```

near the top, just above (or below) the similar lines that ensure `%_udevrulesdir` and `%tmpfiles_create` have sane values?